packages/hash/frontend/src/blocks/page/ComponentView.tsx

```diff
@@ -92,6 +93,7 @@ export class ComponentView implements NodeView<Schema> {
     private readonly getPos: () => number | undefined,
     private readonly renderPortal: RenderPortal,
     private readonly meta: HashBlockMeta,
+    private readonly schema: JsonSchema,
```

Comment:
When I started writing this, `meta` contained both the block metadata and the block schema. #872 removed the schema from it. I need the schema, so I've put it back here, but if there's a better alternative to doing this please let me know cc @nathggns

Reply:
I think the tidier solution is to replace both with `HashBlock` and just have a single `block` parameter.

Reply:
Turns out I did this in https://github.com/hashintel/hash/pull/883 (but closed that, so you can copy that)